Xamarin.Forms.ControlGallery.Android/FormsAppCompatActivity.cs

```diff
@@ -41,12 +41,10 @@ protected override void OnCreate(Bundle bundle)
 			base.OnCreate(bundle);
 
 #if TEST_EXPERIMENTAL_RENDERERS
-			// CollectionView lets us test CollectionView stuff until it's officially released
-			Forms.SetFlags("FastRenderers_Experimental"/*, "CollectionView_Experimental", "Shell_Experimental"*/); 
-#else
 			// Fake_Flag is here so we can test for flag initialization issues
-			// CollectionView lets us test CollectionView stuff until it's officially released
-			Forms.SetFlags("Fake_Flag"/*, "CollectionView_Experimental", "Shell_Experimental" */); 
+			Forms.SetFlags("Fake_Flag"/*, "CollectionView_Experimental", "Shell_Experimental"*/); 
```

Comment:
Why did `Fake_Flag` get moved?

Reply:
The idea is that we'll always set a flag of some sort. Since I removed the flag from TEST_EXPERIMENTAL_RENDERERS and added a flag when we're not on fast renderers, I just moved the fake flag.

Reply:
Oh, I see.